conf/messages.en

```diff
@@ -90,49 +90,47 @@ registerBusinessRepresenting.error.field2.required = Enter field2
+enterContactDetails.heading = Who is the contact for this application?
 enterContactDetails.field1 = Full name
 enterContactDetails.field2 = Email address
-enterContactDetails.field3 = Direct telephone number
+enterContactDetails.field3 = Telephone number (optional)
 enterContactDetails.checkYourAnswersLabel = Contact details
 enterContactDetails.error.field1.required = Enter your full name
-enterContactDetails.error.field2.required = Enter email address
-enterContactDetails.error.field3.required = Enter direct telephone number
-enterContactDetails.error.field2.invalid = Enter a valid email address
+enterContactDetails.error.field2.required = Enter an email address
+enterContactDetails.error.field3.required = Enter a telephone number
+enterContactDetails.error.field2.invalid = Enter an email address in the right format
 enterContactDetails.error.field1.length = Full name must be 100 characters or less
 enterContactDetails.error.field2.length = Email address must be 100 characters or less
 enterContactDetails.error.field3.length = Direct telephone number must be 20 characters or less
 
 informationAboutYourItem.title = Provide information about your item
 informationAboutYourItem.heading = Provide information about your item
-informationAboutYourItem.legend = Is there any information about your item that you do not want to be published on the commodity code ruling website?
-informationAboutYourItem.yesIHaveInfo = Yes - I have some information about my item that I do not want published
-informationAboutYourItem.no = No - all information about my item can be published
+informationAboutYourItem.legend = Is there any information about your item that you do not want to be published?
+informationAboutYourItem.yesIHaveInfo = Yes
+informationAboutYourItem.no = No
 informationAboutYourItem.checkYourAnswersLabel = Information about your item
-informationAboutYourItem.error.required = Select if you have information about your item
-informationAboutYourItem.listItem1 = Confidential information
-informationAboutYourItem.listItem2 = Describe your item
-informationAboutYourItem.listItem3 = Upload photographs, manuals and other supporting information
-informationAboutYourItem.listItem4 = Commodity code (optional)
-informationAboutYourItem.listItem5 = When to send a sample to the classification team
+informationAboutYourItem.error.required = Select yes or no
+informationAboutYourItem.listItem1 = any confidential information you do not want to be published on the BTI ruling website
+informationAboutYourItem.listItem2 = a detailed description of your item to help to determine its classfication
+informationAboutYourItem.listItem3 = photographs, manuals, technical specifications and other information to support your description and help to speed up the application
+informationAboutYourItem.listItem4 = a commodity code if you have found one
+informationAboutYourItem.listItem5 = a sample if it is needed
```

Comment:
how about `a sample if needed`? is it easier to read?

Reply:
Ok

Reply:
Always happy to lose words